Parsedown.php

```diff
@@ -75,6 +75,27 @@ function setUrlsLinked($urlsLinked)
 
     protected $urlsLinked = true;
 
+    function setSafeLinksEnabled($safeLinksEnabled)
+    {
+        $this->safeLinksEnabled = $safeLinksEnabled;
+
+        return $this;
+    }
+
+    protected $safeLinksEnabled = true;
```

Comment:
Yes, so some of this is spoken to by decisions made in the original PR.

I think the default was @erusev's suggestion as far as I can tell though, see https://github.com/erusev/parsedown/pull/276#issuecomment-71331724

If it was up to me I'd make `markupEscaped` default to on too 😉

I'm happy to go with the flow here though, so I'll await some more feedback